erpc_c/transports/erpc_uart_cmsis_transport.h

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2014-2016, Freescale Semiconductor, Inc.
  * Copyright 2016-2020 NXP
+ * Copyright 2021 ACRIOS Systems s.r.o.
```

Comment:
this file has not been modified, no copyright update needed :)

Reply:
Another one :D Sorry i will revert that.